src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java

```diff
@@ -33,11 +33,19 @@
 
 	private JButton Play, Instruction, Exit, Resume, Easy, Medium, Hard, Back, Menu, Credit, PlayBGM, PauseBGM, HighScore, NextPage;
 	private JButton Character, CMenu, Kwhale, Shark;
-	private JFrame frame, instruct;
+
 
 //	private int type;
```

Comment:
I know this isn't from your PR, but it's right here...  Can you remove this unused code too?

Reply:
you mean I should remove it in my master branch?

Reply:
Nah - let's take it out in this branch (diffBG).

Reply:
it is not from PR. I don knoe why it is here. I just commited it and hope you can check it again.

Reply:
i can not push it rn, wait for few minutes